pkg/test/local/environment.go

```diff
@@ -18,10 +18,17 @@ import (
 	"fmt"
 	"testing"
 
+	"istio.io/istio/pilot/pkg/model"
+	envoyv1 "istio.io/istio/pilot/pkg/proxy/envoy/v1"
 	"istio.io/istio/pkg/test/dependency"
 	"istio.io/istio/pkg/test/environment"
 	"istio.io/istio/pkg/test/fakes/policy"
 	"istio.io/istio/pkg/test/internal"
+	"istio.io/istio/pkg/test/local/pilot"
+)
+
+const (
+	namespace = "istio-system"
```

Comment:
This is not used yet, but can you add a TODO to consume this from the environment member?
https://github.com/istio/istio/blob/0ef0df56d4af572d1fe9fb94eccf5dd0c6c679c3/pkg/test/cluster/environment.go#L34